src/components/Home/index.js

```diff
@@ -1,14 +1,24 @@
 import React from 'react';
-import ContainerOne from './ContainerOne';
 import styles from './styles.module.scss';
-import ContainerTwo from './ContainerTwo';
+
 
 function Home() {
     return(
         <div className={styles.home}>
-            <ContainerOne/>
+            <div className={styles.cont1}>
+                <div className={styles.containerHome}>
+                    <h1 className={styles.title}>¡Hola!</h1>
+                    <strong>Queremos darte una mano :)</strong><br/>
+                    <strong>¿Cómo podemos ayudarte hoy?</strong>
+                </div>
+            </div>
             <hr className={styles.hr}/>
-            <ContainerTwo/>
+            <div className={styles.cont2}>
+                <div className={styles.containerButton}>
+                <button className={styles.button1}>QUIERO DONAR</button>
+                <button className={styles.button2}>QUIERO PEDIR</button>
+                </div>
+            </div>
```

Comment:
Presta atención al indentado. Recuerda pasarte por la [guía de estandarización de código](https://docs.google.com/document/d/1gJ0gB1NkinQXyRe5dk71yMUYE8c6dupLc-lYQ0osSro) para ver cómo hacerlo.
```suggestion
                <div className={styles.containerButton}>
                    <button className={styles.button1}>QUIERO DONAR</button>
                    <button className={styles.button2}>QUIERO PEDIR</button>
                </div>
            </div>
```

Reply:
Instale el EsList, y lo revise, se supone que ahora esta corregido